services/adgs/config/adgs_ws_config.yaml

```diff
@@ -103,6 +103,7 @@ adgs:
         wrapper: '"{}"'
         operations:
             and:
+              - "contains(Name, '{Name}')"
```

Comment:
This will need to be put in my new template file instead, but you can merge and I will do it in my branch